spec/features/general_feedback_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+RSpec.feature 'Giving general feedback for the service' do
+  before(:each) { visit new_feedback_path }
+
+  it 'displays the general feedback page' do
+    expect(page).to have_content('Give feedback')
```

Comment:
Minor: Not blocking this PR, it's nicer to use the internationalised versions of these strings, to reduce flakiness when the translation string fails.

Reply:
Sure, I changed it at the last moment.
(Wasn't sure if the internationalised version helped with the readability of the test at first glance.)